var/spack/repos/builtin/packages/care/package.py

```diff
@@ -186,7 +186,7 @@ def initconfig_compiler_entries(self):
         compiler = self.compiler
         entries = super().initconfig_compiler_entries()
 
-        if spec.satisfies("+rocm"):
+        if spec.satisfies("+rocm ^blt@:0.6.99"):
```

Comment:
You can do:
```suggestion
        if spec.satisfies("+rocm ^blt@:0.6"):
```
to mean _any_ v0.6